### ISSUE TITLE
Issues with sails-hook-sockets and 0.12.0-rc6

### ISSUE THREAD

**Issue by ryanwilliamquinn** [balderdashy/sails#3496]
When I run my tests I get this error:
```
error: A hook (`orm`) failed to load!
...
1)  "before all" hook:
     Uncaught TypeError: Cannot read property 'subscribe' of undefined
      at node_modules/sails/node_modules/sails-hook-sockets/lib/connect-to-admin-bus.js:33:43
      at prepareDriver (node_modules/sails/node_modules/sails-hook-sockets/lib/prepare-driver.js:27:16)
      at connectToAdminBus (node_modules/sails/node_modules/sails-hook-sockets/lib/connect-to-admin-bus.js:22:5)
      at node_modules/sails/node_modules/sails-hook-sockets/lib/prepare-adapter.js:69:16
      at prepareDriver (node_modules/sails/node_modules/sails-hook-sockets/lib/prepare-driver.js:27:16)
      at node_modules/sails/node_modules/sails-hook-sockets/lib/prepare-adapter.js:42:7
      at _getAdapter (node_modules/sails/node_modules/sails-hook-sockets/lib/prepare-adapter.js:148:12)
      at prepareAdapter (node_modules/sails/node_modules/sails-hook-sockets/lib/prepare-adapter.js:33:5)
      at whenReady (node_modules/sails/node_modules/sails-hook-sockets/lib/initialize.js:107:7)
      at node_modules/sails/lib/app/private/after.js:91:14
      at node_modules/sails/node_modules/async/lib/async.js:721:13
      at node_modules/sails/node_modules/async/lib/async.js:52:16
      at async.forEachOf.async.eachOf (node_modules/sails/node_modules/async/lib/async.js:236:30)
```
If I install sails-hook-sockets at 0.12.x instead of 0.13.x, everything works fine.

**Comment by sgress454** [comment]
Thanks @ryanwilliamquinn -- can you try installing v0.13.4 and confirm that the issue is fixed?

**Comment by mikermcneil** [comment]
@ryanwilliamquinn

```bash
npm install sails-hook-sockets@latest --force
```

**Comment by ryanwilliamquinn** [comment]
Thanks for the replies, this is working now as of this fix in sails-hook-sockets (0.13.4):
https://github.com/balderdashy/sails-hook-sockets/commit/14210dc8d81e638f198493e05dda5eb8651f0e8f
